fmriprep/interfaces/images.py

```diff
@@ -124,6 +124,27 @@ def _run_interface(self, runtime):
         return runtime
 
 
+class PruneExcessiveZoomInputSpec(BaseInterfaceInputSpec):
+    t1w_list = InputMultiPath(File(exists=True), mandatory=True,
+                              desc='input T1w images')
+    max_scale = traits.Float(3.0, usedefault=True,
+                             desc='Maximum scaling factor in images to accept')
+
+
+class PruneExcessiveZoomOutputSpec(TraitedSpec):
+    t1w_dropped_list = OutputMultiPath(exists=True, desc='invalid T1w images')
+    t1w_valid_list = OutputMultiPath(exists=True, desc='valid T1w images')
```

Comment:
Yes! Initially, it was to be used for a posterior reporting, but I ended adding reporting in this same interface. I'll remove it for now.